source/fe/fe_enriched.cc

```diff
@@ -584,7 +586,7 @@ FE_Enriched<dim, spacedim>::fill_fe_face_values(
 
   if (is_enriched)
     multiply_by_enrichment(
-      quadrature, fe_data, mapping_data, cell, output_data);
+      quadrature[0 /*TODO*/], fe_data, mapping_data, cell, output_data);
```

Comment:
Since we use the index `0` here, can you add the `Assert` like in the other functions?